### ISSUE TITLE
table pingpong

### ISSUE THREAD

**Issue by khodjas** [bdeecho/Meta#29]
Mise en place de Tables de ping – pong
# Pourquoi ?

Dynamiser le campus et attribuer aux étudiants divers moyens de divertissement au besoin !
# Ou ?

*Une intérieur : hall principal (au niveau du piano ? /  à droite de l’entrée (en face, à gauche) ? /  en face de l’ascenseur du bâtiment B?)

*Une extérieur : à cote de l’entrée principale (en bas des escaliers ?) / sur le parvis ? / le couloir à la sortie du RU ? 
# Comment ?

*Fixé au sol !
*S’adresser à la mairie ainsi qu’à la vie associative

**Comment by arya-coding** [comment]
Il faut plus creuser, c'est à dire trouver les produits et les entreprises qui pourrait nous les fournir. 
Il ne suffit pas de penser au projet, mais de vraiment chercher toutes les informations, car c'est à nous de le faire, la vie associative et la mairie ne le feront pas.

Soutenant le projet et souhaitant aider Sofiane, je suggère cela :

La boutique est décathlon. (GoSport n'est malheureusement pas aussi fourni en matériel de tennis de table que son concurrent).
Pour l'intérieur de l'Université, je recommande celle-ci :

![capturqsdqsde](https://cloud.githubusercontent.com/assets/9800251/18629861/d0d737ce-7e68-11e6-8bfa-f619324e0c1e.PNG)
http://www.decathlon.fr/cornilleau-510-pro-outdoor-g-id_8352846.html

Le prix peut paraître important, mais la table est conçu pour le zoo qu'est l’Université. Pas de filet, résistante aux chocs, aux assises de personnes malveillantes et autres détérioration.s
Elle peut être fixé au sol.

Il faudrait aussi qu'Echo se fournisse en raquette et balle.
Je propose les équipements suivants pour une durée d’utilisation optimale.

Deux boîtes de 6 balles classique, pas besoin de balle particulière.
![ca222pture](https://cloud.githubusercontent.com/assets/9800251/18630298/df92bb1e-7e6b-11e6-8209-f0d6e2670a39.PNG)
http://www.decathlon.fr/artengo-fb850-x6-blanc-orange-id_8160177.html

Pour les raquettes, x6 :
http://www.decathlon.fr/artengo-fr700-id_8286996.html
![captuqsdqsre](https://cloud.githubusercontent.com/assets/9800251/18630338/3d79e5c2-7e6c-11e6-9e94-f541aeb7796a.PNG)

Pour l'extérieur, je pense que devant l'entrée principale c'est impossible car c'est une entrée pompier et urgentiste.
Il faudra l'installer dehors à l'étage 1 du bâtiment A. Mais l'installation ne sera pas facile car elle pèse 1 à 2 tonnes la table et qu'il y a une hauteur limité due au bâtiment A. Il faudra surement louer un engin de chantier ou trouver une solution pour l'installer. 
Et la table coûte entre 200 à 2000€, (Moyenne 1200€). Il faudrait voir avec la vie associative.

http://www.direct-collectivites.com/tables-de-ping-pong/9669-table-de-ping-pong-beton-design-plateau-brut.html?gclid=CjwKEAjwmf6-BRDi9fSN7Ijt1wUSJAASawcjlo2u1fqNGyrCjTuTEV3Jv-izAquqzJrhuSxf6E_AdBoCoz7w_wcB
![capture](https://cloud.githubusercontent.com/assets/9800251/18630232/62c46fd8-7e6b-11e6-8963-a5077adfdd43.PNG)

Ce que je suggère, c'est que pour la table à l'extérieur, on en parle à la vie associative.
Pour la table à l'intérieur, voilà le budget pour la table en intérieur (@AlexFourgs , peut t'on la mettre dans la demande de subventions ?)

![cqsdfqsdfapture](https://cloud.githubusercontent.com/assets/9800251/18630380/893812a4-7e6c-11e6-8ebe-bfbc878a8116.PNG)

**Comment by AlexFourgs** [comment]
Ce serait intéressant de proposer cela avec le soutien des autres BDE de St Martin, un projet en commun quoi.

**Comment by arya-coding** [comment]
@AlexFourgs Je ne suis pas d'accord pour les raisons suivantes :
- Le prix est raisonnable.
- L'idée est de Sofiane, membre d'Echo.
- J'ai soutenu l'idée et bossé dessus.

Pourquoi leur mettre le projet entre les mains alors que c'est nous qui avons travaillé dessus ?

**Comment by Uinelj** [comment]
Je pense que c'est du matériel que l'université devrait acheter, plutôt que nous. Cependant, on peut leur proposer un projet construit et cohérent, où ils n'auraient qu'à "sortir la CB".

**Comment by AlexFourgs** [comment]
@Uinelj faudrait en discuter avec la Fédé sinon pour voir qui doit sortir les thunes

**Comment by Uinelj** [comment]
Ca reste un agrément pour TOUS les étudiants, qui sera à disposition dans la Rotonde, je vois pas pourquoi ça serait à Echo de sortir les thunes :/
Pour le baby comment ça s'était passé ?

**Comment by arya-coding** [comment]
Mais le budget ping pong sera demandé lors de la demande de subventions, et eux prendront en compte que c'est pour TOUS les étudiants, et non pas à ECHO ?

**Comment by AlexFourgs** [comment]
@aryus96 Exactement. Du coup on pourra ajouter ça à la demande de subvention et si il manque des thunes, tous les BDE allongent de manière égale ce qu'il manque.

**Comment by khodjas** [comment]
Merci Arya pour tes recherches.. 
C'est parfait ce que tu propose c'est exactement a quoi je pensais ! 
Et sinon faudrait qu'on se mette d'accord du coup si on en parle aux autres BDE ou pas ?

**Comment by AlexFourgs** [comment]
Ouai !

Du coup faudrait lister les BDE présents sur St Martin :
- Hazybot
- Remed
- Univer'Nature
- Echo
- Superowl
- Phenix 

Auquel on peut ajouter le soutien de la Fédération UCP sans aucun doute !

**Comment by arya-coding** [comment]
Franchement je ne suis pas chaud pour rajouter les autres BDE. J'ai l'impression de leur donner "le fruit de notre travail".

Pour la table en béton, je me suis renseigné, sur les dalles c'est pas possible (à priori), il faudrait voir si c'est enviseagable sur du bitume

**Comment by Eresia** [comment]
[HS] Faudrait arrêter d’appeler toutes les assos des BDE ça n'a plus de sens[/HS]

**Comment by arya-coding** [comment]
@khodjas Tu en as parlé avec la vie associative ?
Car du coup, ce n'est pas dans la demande de subvention non @AlexFourgs ?

J'ai un peu le seum d'avoir travaillé sur un projet qui ne risque pas d'être réaliser (du moins, quand je suis en France).

**Comment by khodjas** [comment]
je savais pas que j'étais censé leur parler pour ça. 
Sinon pourquoi ça le serais pas sur la demande de subvention ?

**Comment by AlexFourgs** [comment]
Parce que en soit ça doit être un projet proposer par Echo à l'Université.

On devrait en parler avec les autres associations, et présenter un dossier qui met en avant qu'Echo (Sofiane Kohdja) est à l'origine de l'initiative et que les autres associations nous soutiennent.

Ensuite pour moi c'est à la fac de le faire.
